data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/parse/AbstractParseProcessor.java

```diff
@@ -100,7 +110,10 @@ public Collection<Record<Event>> doExecute(final Collection<Record<Event>> recor
                     event.delete(this.source);
                 }
             } catch (Exception e) {
-                LOG.error(EVENT, "An exception occurred while using the {} processor on Event [{}]", getProcessorName(), record.getData(), e);
+                processingErrorsCounter.increment();
```

Comment:
I think this is a bit broad. This metric could occur if we have a problem that is not necessarily a parsing error (e.g. `event.delete()`.

Related to my comment above, this might work better if we split the exceptions into two kinds.